test/unit/logic/transaction_pool.js

```diff
@@ -1223,52 +1249,6 @@ describe('transactionPool', () => {
 		});
 	});
 
-	describe('expireTransactions', () => {
```

Comment:
Why removed those tests?

Reply:
@4miners Their won't be any invalid transactions, so I removed the unwanted tests.

Reply:
They look moved, not removed.